lib/Runtime/Library/TypedArray.cpp

```diff
@@ -606,15 +606,39 @@ namespace Js
             object;
     };
 
-    BOOL TypedArrayBase::HasProperty(PropertyId propertyId)
+    BOOL TypedArrayBase::HasOwnProperty(PropertyId propertyId)
+    {
+        return HasPropertyEx(propertyId) == Property_Found;
+    }
+
+    inline BOOL TypedArrayBase::CanonicalNumericIndexString(PropertyId propertyId, ScriptContext *scriptContext)
+    {
+        PropertyString *propertyString = scriptContext->GetPropertyString(propertyId);
```

Comment:
Seems kind of wasteful to create a property string just for this check. But I guess this is only limited to if people ask for float properties.......